http_check/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - http_check
 
+1.2.0 / 2017-08-18
+==================
+
+### Changes
+
+* [FEATURE] Add support for client side certificate.
```

Comment:
Could you add a link to this PR (with the link in the footer). You can see examples in the rest of the file.